addons/l10n_in/data/template/account.fiscal.position-in.csv

```diff
@@ -1,28 +1,4 @@
 "id","name","tax_ids/tax_src_id","tax_ids/tax_dest_id"
-"fiscal_position_in_inter_state","Inter State","sgst_sale_1","igst_sale_1"
-"","","sgst_sale_2","igst_sale_2"
-"","","sgst_sale_5","igst_sale_5"
-"","","sgst_sale_12","igst_sale_12"
-"","","sgst_sale_18","igst_sale_18"
-"","","sgst_sale_28","igst_sale_28"
-"","","sgst_purchase_1","igst_purchase_1"
-"","","sgst_purchase_2","igst_purchase_2"
-"","","sgst_purchase_5","igst_purchase_5"
-"","","sgst_purchase_12","igst_purchase_12"
-"","","sgst_purchase_18","igst_purchase_18"
-"","","sgst_purchase_28","igst_purchase_28"
-"fiscal_position_in_export","Export","sgst_sale_1","igst_sale_1"
```

Comment:
This will need to be archived manually for people who update the chart template on existing databases